lib/eslint/eslint.js

```diff
@@ -157,6 +158,7 @@ function processOptions({
     baseConfig = null,
     cache = false,
     cacheLocation = ".eslintcache",
+    cacheStrategy = "metadata",
```

Comment:
We should add `cacheStrategy` to `return { ... }`.

Otherwise, it has no effect as it won't be passed to `CLIEngine` [here](https://github.com/eslint/eslint/blob/1d11d48a41f02fed627a37072f1fa1d7bd0dca2d/lib/eslint/eslint.js#L431). I tested `--cache-strategy content` on CLI and it still stores `mtime` and `size` instead of `hash` in the cache file.

Reply:
You're absolutely right, this has been fixed :)